css/style.css

```diff
@@ -1335,6 +1336,47 @@ div#mobile-top, div#mobile-search, img#mobile-hamburger
 	display: none;
 }
 
+.page-contents
+{
+    width: 300px;
+    padding-right: 10px;
+    border: solid black;
+    border-width: 2px 0;
+    font-size: 12px;
+    text-align: left;
+    float: right;
+    margin: 0 0 2em 2em;
+}
+
+.page-contents-header
+{
+	text-align: center;
+}
+
+.page-contents-header h3
+{
+	display: inline;
+	color: #444;
+	font-weight: bold;
+	margin-top: 0;
+}
+
+.page-contents li
+{
+	line-height: 1.3;
+	padding-bottom: 0 !important;
```

Comment:
Instead of taking out the `!important` sledgehammer, I suggest making the selector more specific so that it beats the other one. The other one is `div#content li`, so `div#content .page-contents li` should get us there.